### DISCUSSION TITLE
Looking for menu customization guide

### DISCUSSION THREAD

**Discussion by JustLew3** [bigtreetech/BIGTREETECH-TouchScreenFirmware#1708] [Q&A]
I've recently purchased a Mingda D2 and have been trying to edit the menus in Marlin. I'm looking for some guidance on adding buttons and/or sub menus for the TFT. Without the Marlin simulator, my Marlin menu tweaks do not translate onto TFT. My main goal is to add a "Level Corners" button, but I'd also like to learn how to modify menus.

**Comment by oldman4U** [comment]
Hi.

Level corners is available in touchscreen mode. For everything else you need very good development skills and a lot of time.

Hope this helps

**Reply by JustLew3** [reply]
Understood. I meant to offense to real software engineers, I was merely looking for a place to start. I've found several guides to Marlin which allowed to get my feet wet with the boards, but nothing on the screens. Over the winter I built a V1 MPCNC and caught the bug by bending the SKR pro to my will!

**Comment by oldman4U** [comment]
Unfortunately what you would like to do is a lot of work and there are
already some tickets related to it.

Let’s see what happens with this discussion.

👍🏻

JustLew3 <notifications@github.com> schrieb am Do. 11. März 2021 um 06:23:

**Comment by jafarniknam** [comment]
Hi . same question . I need to delete some menu Items and add another features for my six axes robot (not a 3D printer) . 
were you able to do that ? if yes can you help me How to do that ? 
thanks a lot
